Applied versioning code to set path in com_api (resource.php) library…

amolkisan opened the pull request:
… itself

This is to set resource path if there is version provided in API url.

coolbung commented:
Can you apply code formatting and make sure the code passes PHPCS validation. Also the type for the version should be float, currently is CMD (default)

amolkisan commented:
I have added only 4 lines and they are well formatted. I didn't get what you want me to do for code formatting. Used online code sniffer, that doesn't give me major errors. Also when I am using type FLOAT then if version contains value 1.0, it returns only 1, So I removed it.

thite-amol commented:
Closing this PR. New discussion is opened under https://github.com/techjoomla/com_api/issues/64

cc
@coolbung
